tools/coqmod/dune

```diff
@@ -0,0 +1,11 @@
+(executable
+ (name coqmod)
+ (public_name coqmod)
+ (package coq-core)
+ (modes exe)
```

Comment:
why do you add `modes`?

Reply:
I copied it from coqdep which had byte too, I will get rid of it.